### PR TITLE
Docs for 0.27

### PR DESCRIPTION
**Pull request by mazameli**
Alerts and x-ray insights. Still need to add images, and something about premium embedding.

### CODE REVIEW

File: docs/users-guide/14-x-rays.md [hunk 1]
```diff
@@ -44,3 +46,25 @@ The comparison report shows how many rows there are in the segment versus the pa
...
+
+![Insights](./images/insights.png)
+
+Here's some of the things Metabase might point out to you:
+- The normal range of values in your data.
+- Whether or not there seem to be a lot of missing values or zeros in your data.
+- Whether or not your data has an overall trend, and if so, what kind of trend.
+- Whether or not your data can be broken up into different distinct stages of growth or decline.
+- Whether or not your data is stationary.
+- Whether or not your data has a strong or mild autocorrelation [@sb, I don't totally understand this one. Something to do with lagged repeating patterns?]
```
Comment (kdoh) [inline docs/users-guide/14-x-rays.md]: Should probably @sbelak here for help.

**Review by kdoh — CHANGES_REQUESTED** [review]
Looks like there are still some notes left in the insights docs that we probably don't want to merge. Other than that this looks good.

File: docs/users-guide/14-x-rays.md (hunk 1 above)
Comment (mazameli) [reply docs/users-guide/14-x-rays.md]: Yep, thanks for catching that. @sbelak, there were a couple of statistical terms in here that I couldn't explain easily from context or light Wikipedia-ing, so your input here would be delightful.

File: docs/users-guide/15-alerts.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+
+## Getting alerts about questions
```
Comment (salsakran) [inline docs/users-guide/15-alerts.md]: we should also add this page to the index/readme of the users guide

File: docs/users-guide/14-x-rays.md (hunk 1 above)
Comment (salsakran) [reply docs/users-guide/14-x-rays.md]: Not sure we need to specify these out entirely. I expect them to change fairly often, and this feels a bit too specific, and nerdy. If anything we should be changing the insights copy to english vs using statistical terms.

**Review by salsakran — CHANGES_REQUESTED** [review]

File: docs/users-guide/15-alerts.md (hunk 2 above)
Comment (mazameli) [reply docs/users-guide/15-alerts.md]: Yep, good catch.

**Comment by mazameli** [comment]
K, I've vastly cut down the insights bit to just give a hand-wavy overview instead of listing each and every one, and I added alerts to the users guide table of contents. Should be good to go now.

File: docs/administration-guide/13-embedding.md [hunk 3]
```diff
@@ -65,3 +65,7 @@ Dashboards are a fixed aspect ratio, so if you'd like to ensure they're automati
 
 ### Reference applications
 To see concrete examples of how to embed Metabase in applications under a number of common frameworks, check out our [reference implementations](https://github.com/metabase/embedding-reference-apps) on Github.
+
+
+## Premium embedding
+If you'd like to embed Metabase dashboards or charts in your application without the "Powered by Metabase" attribution, you can purchase premium embedding from the Metabase store. [Find out more here](http://store.metabase.com/).
```
Comment (salsakran) [inline docs/administration-guide/13-embedding.md]: Not sure where this link should go, probably a blog post or a propaganda page. @kdoh do you think landing on the product page makes the most sense or should we have a different spot?

**Review by salsakran — APPROVED** [review]
LGTM

File: docs/administration-guide/13-embedding.md (hunk 3 above)
Comment (kdoh) [reply docs/administration-guide/13-embedding.md]: Product page probably makes the most sense.

**Pull request merged**